contrib/doc-gen/generate.go

```diff
@@ -158,6 +159,7 @@ func open(path string) io.Writer {
 }
 
 func run(args ...string) string {
+	args = append([]string{BITCOIN_CHAINOPTION}, args...)
 	out, err := exec.Command(BITCOIN_COMMAND, args...).CombinedOutput()
 	if err != nil {
 		log.Fatalf("Cannot run bitcoin-cli: %s, is bitcoind running?", err.Error())
```

Comment:
```suggestion
		log.Fatalf("Cannot run bitcoin-cli: %s, is bitcoind (regtest) running?", err.Error())
```
I think it would make sense to remind here that it's not enough to run `bitcoind` but it must also run on regtest.

Reply:
Good idea, done.